CMakeLists.txt

```diff
@@ -2,8 +2,6 @@ PROJECT(PlusLib)
 
 CMAKE_MINIMUM_REQUIRED(VERSION 3.3.0)
 
-SET(CMAKE_CXX_STANDARD 11)

```

Comment:
Maybe keep this with the same `IF(NOT DEFINED ...` structure?

Reply:
See potential implementation here: https://github.com/Slicer/ChestImagingPlatform/commit/bc173fdba7db8ebac9119bcef10dffaac20f7f03

Reply:
I've updated this to match how I implemented it in PlusBuild (based on Slicer CIP)